packages/mobile/src/screens/chat-screen/ChatScreen.tsx

```diff
@@ -295,9 +296,9 @@ export const ChatScreen = () => {
     })
     // Need to subtract spacing(2) to account for padding in message View.
     messageTop.current = messageY - spacing(2)
-    setPopupChatIndex(index)
+    setPopupMessageId(id)
     setShouldShowPopup(true)
-  }
+  }, [])
```

Comment:
are we voluntarily using an empty deps array here? what if `itemsRef` updates?

Reply:
Interesting q.. So adding a dependency to `itemsRef.current` is definitely wrong. Adding dep of `itemsRef` is maybe correct, it would only trigger when we set the pointer to something other than the ref it was initialized with at the top? But since it's a const I don't think this is possible.. Let me know if this makes sense @sddioulde.

Reply:
hmm interesting. that makes sense to me thx